packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -2436,4 +2463,17 @@ https://developer.litprotocol.com/v3/sdk/wallets/auth-methods/#auth-method-scope
       // },
     },
   };
+
+  private estimateGasWithMargin = async (
```

Comment:
we have a new naming convention for our code space to have all private methods prefixed with `_`

```suggestion
  private _estimateGasWithMargin = async (
```